benchexec/tools/nacpa.py

```diff
@@ -0,0 +1,92 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+from benchexec import result
+from benchexec.tools import template
+
+
+class Tool(template.BaseTool2):
+    """
+    This tool-info module runs nacpa, a parallel portfolio
+    of natively compiled CPAchecker instances.
+    """
+
+    REQUIRED_PATHS = ["bin", "lib"]
+
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("nacpa", subdir="bin")
+
+    def version(self, executable):
+        return self._version_from_tool(
+            executable, "--version", line_prefix="nacpa version"
+        )
+
+    def program_files(self, executable):
+        return self._program_files_from_executable(
+            executable, self.REQUIRED_PATHS, parent_dir=True
+        )
+
+    def name(self):
+        return "nacpa"
+
+    def cmdline(self, executable, options, task, rlimits):
+        if "--spec" not in options and task.property_file:
+            options = options + ["--spec", task.property_file]
+
+        if (
+            "--data-model" not in options
+            and isinstance(task.options, dict)
+            and task.options.get("language") == "C"
+        ):
+            data_model = task.options.get("data_model")
+            if data_model:
+                options = options + ["--data-model", data_model]
+
+        return super().cmdline(executable, options, task, rlimits)
```

Comment:
Does your tool support tasks with more than one or no input file? If not, this should be changed here to not (indirectly) use `task.input_files_or_identifier`.